instance/tasks.py

```diff
@@ -339,3 +344,117 @@ def cleanup_old_betatest_users():
         users_to_delete = queryset.filter(is_active=False, profile__modified__lte=delete_cutoff)
         logger.info('Deleting %s users.', users_to_delete.count())
         users_to_delete.delete()
+
+
+class KillZombiesRunner:
+    """
+    Helper class to run `kill_zombies_periodically`
+    """
+
+    def __init__(self):
+        self.region: str = settings.OPENSTACK_REGION
+        self.recipient: list = [email for _, email in settings.ADMINS]
+        self.threshold: int = getattr(settings, "KILL_ZOMBIES_WARNING_THRESHOLD", 10)
```

Comment:
@spokerman12, is there a reason for using `getattr()` calls everywhere instead of just using `settings.SETTING_NAME`? In this case, the default is already defined in `settings.KILL_ZOMBIE_WARNING_THRESHOLD`.

Reply:
Addressed in previous comments.
The first implementation used "environment variables" as it was discussed.